code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -62,15 +64,9 @@
 
 /mob/living/simple_animal/hostile/asteroid/goliath/revive(full_heal = FALSE, admin_revive = FALSE)//who the fuck anchors mobs
 	if(..())
-		move_force = MOVE_FORCE_VERY_STRONG
-		move_resist = MOVE_FORCE_VERY_STRONG
-		pull_force = MOVE_FORCE_VERY_STRONG
 		. = 1
 
 /mob/living/simple_animal/hostile/asteroid/goliath/death(gibbed)
-	move_force = MOVE_FORCE_DEFAULT
-	move_resist = MOVE_RESIST_DEFAULT
-	pull_force = PULL_FORCE_DEFAULT
 	..()
 
```

Comment:
dont make overrides of procs that only call parent. also move_resist still needs to be set/unset from normal